pyartcd/pyartcd/util.py

```diff
@@ -108,3 +108,28 @@ async def kinit():
         'ocp-build/buildvm.openshift.eng.bos.redhat.com@IPA.REDHAT.COM'
     ]
     await exectools.cmd_assert_async(cmd)
+
+
+async def branch_arches(branch: str, ga_only: bool = False):
+    """
+    Find the supported arches for a specific release
+    :param branch: The name of the branch to get configs for. For example: 'openshift-4.12
+    :ga_only: If you only want group arches and do not care about arches_override.
+    :return: A list of the arches built for this branch
+    """
+
+    logger.info('Fetching group config for %s', branch)
+
+    # Check if arches_override has been specified. This is used in group.yaml
+    # when we temporarily want to build for CPU architectures that are not yet GA.
+    cmd = f"doozer --group={branch} config:read-group --yaml arches_override --default '[]'"
+    _, out, _ = await exectools.cmd_gather_async(cmd)
```

Comment:
No need to load group configs twice. You can call `load_group_config` function to get the group config with assembly overridden in dict format.

Reply:
Updated in 5eb8bae480f5fd5b58a3257e84f45711624d2d2d